pkg/controller/nodeipam/ipam/range_allocator.go

```diff
@@ -111,6 +112,12 @@ func NewCIDRRangeAllocator(client clientset.Interface, nodeInformer informers.No
 		klog.V(0).Info("No Service CIDR provided. Skipping filtering out service addresses.")
 	}
 
+	if secondaryServiceCIDR != nil {
+		ra.filterOutServiceRange(secondaryServiceCIDR)
+	} else {
+		klog.V(0).Info("No Secondary Service CIDR provided. Skipping filtering out secondary service addresses.")
```

Comment:
not really needed?

Reply:
it is following the norms as https://github.com/kubernetes/kubernetes/pull/79386/commits/c3e079035a993c18a36963ba6865a64cb476857b?file-filters%5B%5D=.go#diff-8f965170bb98fe3252b76e74d095a57bR112. I can remove if needed